Packages/ClientRuntime/Sources/Networking/Http/UnknownHttpServiceError.swift

```diff
@@ -19,9 +21,15 @@ public struct UnknownHttpServiceError: HttpServiceError, Swift.Equatable {
 }
 
 extension UnknownHttpServiceError {
-    public init(httpResponse: HttpResponse, message: String? = nil) {
+    public init(httpResponse: HttpResponse, message: String? = nil, errorCode: String? = nil) {
         self._statusCode = httpResponse.statusCode
         self._headers = httpResponse.headers
         self._message = message
+        self._errorCode = errorCode
     }
 }
+
+extension UnknownHttpServiceError: CodedError {
+
+    public var errorCode: String? { _errorCode }
```

Comment:
Can we add docs as to what an error code is (or should be) for an UnknownHttpServiceError?

Reply:
Added